lib/Connection.js

```diff
@@ -29,6 +29,13 @@ function Connection(router, req, res) {
         if (options) {
             if (options.props) request.setProps(options.props);
             if (options.resource) request.setResource(options.resource);
+
+            if (options.query) {
+                for (var key in options.query) {
```

Comment:
You'll have to do some splitting with ```&``` and then ```=``` to get maps of key/value pairs to work with.